apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Snippets/BigQueryClientSnippets.cs

```diff
@@ -528,6 +528,40 @@ public void UploadAvro()
         // See [UploadAvro](ref) for a synchronous example.
         // End see-also
 
+        [Fact]
+        public void UploadParquet()
+        {
+            string projectId = _fixture.ProjectId;
+            string datasetId = _fixture.GameDatasetId;
+            string tableId = _fixture.GenerateTableId();
+
+            var typeInfo = GetType().GetTypeInfo();
+            string resourceName = typeInfo.Namespace + ".us-states.parquet";
+            var stream = typeInfo.Assembly.GetManifestResourceStream(resourceName);
+            // Snippet: UploadParquet(string, string, Stream, *)
+            BigQueryClient client = BigQueryClient.Create(projectId);
+            // This example creates a new table.
+            BigQueryJob job = client.UploadParquet(datasetId, tableId, stream);
+            // Use the job to find out when the data has finished being inserted into the table,
+            // report errors etc.
+            // End snippet
+
+            AssertJobCompletesSuccessfully(job);
+        }
+
+        // See-also: UploadParquet(string, string, Stream, *)
+        // Member: UploadParquet(TableReference, *, *)
+        // Member: UploadParquet(string, string, string, *, *)
+        // See [UploadParquet](ref) for an example using an alternative overload.
+        // End see-also
+
+        // See-also: UploadParquet(string, string, *, *)
```

Comment:
question to understand: Is this adding a "See also ref" to each of the members listed below linking to `UploadParquet(string, string, Stream, *)` or the other way around? I think yes because it says "for a synchronous example" but just wanted to confirm.

Reply:
If you have:

```csharp
// See-also: A
// Member: B
// Member: C
// Member: D
// See [A](ref) for an example.
```

then the docs for B, C and D will contain a link to A. The aim is to remove duplication (and to make it far more feasible to actually provide snippets for at least one method per operation).